frontend/catalyst/autograph/ag_primitives.py

```diff
@@ -570,6 +570,9 @@ def qnode_call_wrapper():
                 return ag_converted_call(fn.func, args, kwargs, caller_fn_scope, options)
 
             new_qnode = qml.QNode(qnode_call_wrapper, device=fn.device, diff_method=fn.diff_method)
+
+            for program in fn.transform_program._transform_program:
```

Comment:
Hi @dime10 , since `Qnode` does not have a way to copy `TransformConainer`s. Is it fine that we disable the protected member check here?

Reply:
In `qml.QNode`, `transform_program` is a public attribute. Maybe the qjit analogue hasn't also made it public?

Reply:
The compiler will report a TypeError if I pass `transform_program` directly. It seems that it only accepts `transform_program._transform_program`. I do not find a qjit counterpart for it.  But I think directly copying qnode instead of creating new one will work. Thank you,